web/src/redux/Store.ts

```diff
@@ -0,0 +1,5 @@
+import { StoreModule } from '@ngrx/store';
```

Comment:
keep it as store/index.ts

Reply:
moved to store folder and put it `index.ts`